### ISSUE TITLE
Jaeger service not reachable from frontend

### ISSUE THREAD

**Issue by oguzcankirmemis** [Open-IoT-Service-Platform/platform-launcher#111]
With this [PR](https://github.com/Open-IoT-Service-Platform/oisp-frontend/pull/81), I am sending trace logs successfully to jaeger-client, but the client does not seem to get it.
Tested with docker.

**Comment by oguzcankirmemis** [comment]
After some intensive looking, I found out that somehow the way Node.js sends the UDP packages are invalid for the port 5775 (this is really weird because in python it actually works like the way you described).
After some try and guess, I found the correct port for Node.js which is 6832. It is also interesting to note that if you use the port 6831, then the services gets logged, but spans do not reach to agent. Anyways, sorry it took a bit long to figure it out because of the poor documentation at the nodejs side of jaeger.
See this [PR](https://github.com/Open-IoT-Service-Platform/oisp-frontend/pull/81) for the version that works with docker.
